dev/PushNotifications/PushNotificationUtility.cpp

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should the utility implementation be in the header file? I would imagine that this file has to be published to be consumed by the LRP. But also, if the LRP compiles before the SDK, wouldn't a compile error happen since we were looking for the utility file not having published yet?

Would dev/Common be the right place for this code?

Reply:
Create a common utility file between LRP and PushNotifications #1481 - That is the reason I have created this bug. Temporarily I have created the files PushNotificationUtility.h/cpp in dev/PushNotifications. As responses might be little longer it should not block the PR from merging atleast in WNP_LRP. By then once we know where these files should move to we can address it as part of a bug. Let me know if you think otherwise. Thanks.

Reply:
@danielayala94 - realized that PushNotifications.vcxitems is a shared folder. So we don't need to create a new common folder and with the suggestions from Howard, we should be able to call into PushNotificationUtility.h/cpp from anywhere within LRP and beyond. Thanks.